Exercise_2_Currency_Ex/README.md

```diff
@@ -24,23 +24,25 @@ Requirements for this exercise:
 
 ## Current Status
 
-2021-09-20:
+Most recent status shown first
```

Comment:
I always liked this format for these things, the most poignant thing is at the top of the stack and the rest can be easily unearthed if desired ⭐